### PR TITLE
[CFL] minimize crashes

### PR DESCRIPTION
**Pull request by jonathanmetzman**
CC @evverx

### CODE REVIEW

File: infra/cifuzz/fuzz_target.py [hunk 1]
```diff
@@ -194,12 +194,30 @@ def fuzz(self, batch=False):
       if is_reportable or self.config.upload_all_crashes:
         fuzzer_logs = result.logs
         testcase_path = self._save_crash(crash)
+        if is_reportable and self.config.minimize_crashes:
+          # TODO(metzman): We don't to minimize unreproducible crashes.
+          # Use is_reportable to decide this even though reportable crashes
+          # are a subset of reproducible ones.
```
[the remark this answers is not in the document]
Comment (jonathanmetzman) [reply infra/cifuzz/fuzz_target.py:200]: I think if upload_all_crashes is set, then only reproducible, novel crashes are minimized.
Requiring novelty is the only actual "bug" here, but it's minor enough that I don't want to mess up the rest of the code in order to get 100% correct behavior.

[the remark this answers is not in the document]
Comment (evverx) [reply infra/cifuzz/fuzz_target.py:200]: Fair enough. Thanks!

One last question. I'm not sure how `engine_impl.minimize_testcase` works but if it's based on `-minimize_crash=1` provided by libfuzzer (where it doesn't keep track of backtraces) I wonder if CFlite can upload both crashes? I don't think it matters much when projects are already stable but if they just start fixing bugs libfuzzer can minimize crashes to a point where they don't trigger original issues.

[the remark this answers is not in the document]
Comment (oliverchang) [reply infra/cifuzz/fuzz_target.py:200]: Re making sure minimize_crash=1 minimizes to the same crash, we can set `ASAN_OPTIONS=dedup_token_length=3`.

We could either set it here in cifuzz, or set this in `engine_impl.minimize_testcase` itself. The latter makes more sense to me and will just require a change to https://github.com/google/clusterfuzz/blob/f358af24f414daa17a3649b143e71ea71871ef59/src/clusterfuzz/_internal/bot/fuzzers/libFuzzer/engine.py#L537. @jonathanmetzman WDYT?

**Review by oliverchang — APPROVED** [review]
nice!

File: infra/cifuzz/fuzz_target.py (hunk 1 above)
[the remark this answers is not in the document]
Comment (jonathanmetzman) [reply infra/cifuzz/fuzz_target.py:200]: Yeah let's set it in ClusterFuzz.

**Pull request merged**